README.md

```diff
@@ -17,15 +17,15 @@ Right now, until we get some gulp scripts in place make sure to run the followin
 * `npm run build`
 * `npm run build-min`
 
-## Using: 
```

Comment:
Mostly whitespace changes in this file. The only meaningful changes are to wrap HTTP method names in backticks and add `PATCH` to the list of methods that allows a payload